pkg/virtualKubelet/forge/meta.go

```diff
@@ -3,7 +3,11 @@ package forge
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 const (
+	// LiqoOutgoingKey is a label to set on all offloaded resources.
 	LiqoOutgoingKey = "virtualkubelet.liqo.io/outgoing"
+	// LiqoRemoteClusterID is a label to set on all offladed resources to identify the remote cluster.
+	LiqoRemoteClusterID = "virtualkubelet.liqo.io/remoteClusterId"
```

Comment:
The "remote" word can be misleading, can we use something like "originClusterID" to indicate that this id is the id of the cluster that created this pod?